### DISCUSSION TITLE
Cannot start maradns-3-5-0036-win32

### DISCUSSION THREAD

**Discussion by wsw70** [samboy/MaraDNS#121] [Q&A] [answered]
I downloaded the latest version of MaraDNS (`maradns-3-5-0036-win32`) and tried to install it in an admin shell:

```
C:\bin\maradns-3-5-0036-win32>Deadwood.exe --install
Problem creating service
```

There are no more messages, neither in the events log nor in the MaraDNS log file. How can I help to debug this?

**EDIT: I fixed that by closing the `Services` window...**

Now there is a new problem:

```
C:\bin\maradns-3-5-0036-win32>net start Deadwood
System error 5 has occurred.

Access is denied.
```

With this one I am stuck. I  checked the account it was running on (`Local Service`) and permissions to the folder for that account (granted)

Windows version: Windows 11 `10.0.22621.2134`

**Comment by wsw70** [comment, marked as answer]
Solved by ensuring rights on folders all up to the root

**Comment by samboy** [comment]
Thank you for solving this. I am still using Windows 10; I understand Windows 11 has different security parameters than Win10. I run MaraDNS/Deadwood as admin in Windows 10 without issue.

**Comment by wsw70** [comment]
> I run MaraDNS/Deadwood as admin in Windows 10 without issue.

I was surprised as well. Not to mention the dreadful first error message (one would have to think a lot before imagining this), the solution is a new thing for me (having moved from Windows 10 recently).
